res_syntax/src/res_diagnostics.ml

```diff
@@ -142,8 +142,7 @@ let printReport diagnostics src =
     match diagnostics with
     | [] -> ()
     | d :: rest ->
-      Res_diagnostics_printing_utils.Super_location.super_error_reporter
-        Format.err_formatter src
+      Super_location.super_error_reporter Format.err_formatter
```

Comment:
As Super_error was cleaned up, now you can use `Location.error_reporter` here now. 😉